content/_changelogs/8.0.0.md

```diff
@@ -0,0 +1,68 @@
+## 8.0.0
+
+_Released 07/19/2021_
+
+**Summary:**
+
+**Breaking Changes:**
+
+**<Icon name="exclamation-triangle" color="red"></Icon> Please read our
+[Migration Guide](/guides/references/migration-guide) which explains the changes
+in more detail and how to change your code to migrate to Cypress 8.0.**
+
+- When running `cypress run` previous to 8.0, some browsers would launch headed
+  while others were launched headless by default. Cypress now runs all browsers
+  during `cypress run` as headless by default. Addresses
+  [#16832](https://github.com/cypress-io/cypress/issues/16832).
+- The default screen size when running a headless browser has been reverted back
+  to 1280x720 pixels (pre 7.0 behavior). Addresses
+  [#16853](https://github.com/cypress-io/cypress/issues/16853).
+- When running the `—headless` Chrome browser via `cypress run`, the device
+  pixel ratio will now be 1 by default, matching the behavior of all other
+  browsers. This behavior can be overridden through the
+  [browser launch API](/api/plugins/browser-launch-api#Override-the-device-pixel-ratio).
+  Addresses [#17375](https://github.com/cypress-io/cypress/issues/17375).
+- Cypress now enforces version checks for browser launching and will error
+  during `cypress run` and not allow opening the browser in `cypress open` when
+  attempting to open unsupported browser versions. Cypress supports Chrome >=
+  64, Firefox >= 86, and Edge >= 79. Addressed in
+  [#17355](https://github.com/cypress-io/cypress/pull/17355).
+- When `any` type value is returned from the last function, then the argument
+  will no longer be wrapped with `JQuery`. Fixes
```

Comment:
I was having a little trouble understanding this note. This text seems more clear to me (but if it's not accurate, we can skip it)

```suggestion
- No longer wrap returned `any` value from Chainable with `JQuery`. Fixes
```

Reply:
Yeah, this is a not great description, I can try to make it clearer. Basically some things were typed as jQuery objects when they are not necessarily jQuery objects